eiffel-syntax-and-usage/activity-linking.md

```diff
@@ -0,0 +1,100 @@
+
+A _fully orchestrated pipeline_ is completely controlled by a dedicated pipeline orchestrator, such as Jenkins Pipeline or Argo Workflows, and has no activities triggered by Eiffel events. A fully orchestrated pipeline is probably often initiated by a source change in some SCM system, and that source change is then propagated to the pipeline orchestrator through some non-Eiffel-event channel (e.g. an Github web hook or a Gerrit stream-event)
+
+None of the scenarios above is probably relevant for most of the Eiffel event users, but rather a combination of the two where a pipeline is often _triggered_ by an Eiffel event, but then an orchestrator deals with controlling (at least parts of) the pipeline. Such triggers could for example be SCM events (e.g. [SCC](../eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md)/[SCS](../eiffel-vocabulary/EiffelSourceChangeSubmittedEvent.md)) or artifact events (e.g. [ArtC](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md)/[ArtP](../eiffel-vocabulary/EiffelArtifactPublishedEvent.md)/[CLM](../eiffel-vocabulary/EiffelConfidenceLevelModifiedEvent.md)).
+
+To handle the different possible scenarios for pipeline execution, multiple link types are defined in the Eiffel protocol to be used to link to and from activity events.
+
+## Link Types Involved
+This section describes the main link types involved in linking activities in pipelines
+
+### CAUSE
+Identifies a cause of the event occurring, in the situations where the cause is an Eiffel event. This link type is not relevant only for EiffelActivity\*Events, but for any Eiffel event that represent an [occurrence](#occurrence) that was caused by an earlier Eiffel event.
+
+__Required:__ No  
+__Legal sources:__ Any  
+__Legal targets:__ Any  
+__Multiple allowed:__ Yes  
+
+### PRECURSOR
+This link type is used to declare *temporal relationships* between activities in a pipeline, i.e. what other activity/activities preceded this activity.
+
+*The image below shows how an activity that precedes another activity is denoted by a PRECURSOR link from the second ActT event back to the first one*
+
+![alt text](./precursor-simple.png "Simple PRECURSOR Example")
+
+An activity could have multiple PRECURSOR activities (also known as "fan-in"), which means that the activity was triggered in serial after a certain group of parallel activities were triggered. This can be used to declare merge points of a group of parallel activities into a succeeding activity.
+
+*The image below shows how three parallel activities are followed by one activity in serial. It is denoted by PRECURSOR links from the ActT event of the last activity to the ActT event of all preceding parallel activities*
+
+![alt text](./precursor-parallel.png "Parallel PRECURSOR Example")
```

Comment:
I agree, but there seems to be a limitation in Inkscape on that regard. I could manually place those arrows in the picture, but then the connections would be lost and it would be a lot harder to make other adjustments later. I'm afraid we'd need to live with this layout as long as we use Inkscape for this picture, unless Inkscape gets such a new feature.